packages/lodestar/test/unit/tasks/interopSubnetsJoiningTask.test.ts

```diff
@@ -7,6 +7,7 @@ import {ChainEvent, IBeaconChain} from "../../../src/chain";
 import {Gossip} from "../../../src/network/gossip/gossip";
 import {InteropSubnetsJoiningTask} from "../../../src/tasks/tasks/interopSubnetsJoiningTask";
 import {WinstonLogger, bytesToInt, intToBytes} from "@chainsafe/lodestar-utils";
+import * as mathUtils from "@chainsafe/lodestar-utils/lib/math";
```

Comment:
This type of stubs is preventing us from upgrading to Typescript v4. Could you please find a different way to test this without them?

Reply:
@dapplion could you send me the breaking change of Typescript v4 regarding this? I notice this patterns happen at a lot of places in our code.

Reply:
is it ok to do `export * as mathUtils from "./math";` in lodestar-utils and `import {mathUtils} from "@chainsafe/lodestar-utils"` in Typescript v4?